source/daplink/daplink.h

```diff
@@ -19,6 +19,16 @@
  * limitations under the License.
  */
 
+/*******************************************************************************
+* Copyright 2019, Cypress Semiconductor Corporation
```

Comment:
Please make these changes to all of the added copyright blocks on common files.
- Move the Cypress copyright to just below the Arm copyright.
- Remove the rest of the additional copyright block, including the description of changes. We don't list individual changes per copyright owners, as it quickly gets out of sync or becomes unmanageable.

Sorry to took a while to make this comment, I needed to double check our copyright management policy. Thanks!

Reply:
The copyright block has been changed according to your comment